cpp/order/LocalWl.cc

```diff
@@ -34,13 +34,13 @@ void LocalWl::computeYlm(const float theta, const float phi, std::vector<std::co
     for (typename fsph::PointSPHEvaluator<float>::iterator iter(sph_eval.begin_l(m_l, 0, false));
          iter != sph_eval.end(); ++iter)
     {
-        Ylm[(j + m_l) % (2 * m_l + 1)] = *iter;
+        // Manually add the Condon-Shortley phase, (-1)^m, to positive odd m
+        float phase = (j % 2 == 1) ? -1 : 1;
+        Ylm[m_l + j] = phase * (*iter);
```

Comment:
Do we not need the old modulo math? It would make sense that we just get the positive values from the iterator, but I'm confused about why the modulus used to be there at all.

Reply:
No, that modulus was unneeded. I tried testing with and without it, and it made no difference.

Reply:
Yeah that sounds right. Another random old oversight I suppose.